src/lib/pickles/wrap_verifier.ml

```diff
@@ -617,8 +751,105 @@ struct
         absorb sponge PC (Boolean.true_, x_hat) ;
         let w_comm = messages.w_comm in
         Vector.iter ~f:absorb_g w_comm ;
+        let joint_combiner =
+          match messages.lookup with
+          | None ->
+              Types.Opt.None
+          | Maybe (b, l) ->
+              failwith "TODO"
```

Comment:
This `TODO` guy again. Hope you are substituted by a more thorough string very shortly :-)